pack/buildpack/bin/build

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+echo "---> Functions Framework C++ Buildpack"
+
+layers="$1"
+
+echo "-----> Setup vcpkg cache"
+export VCPKG_DEFAULT_BINARY_CACHE=${layers}/vcpkg
+export VCPKG_OVERLAY_PORTS=/layers/cpp/gcf/vcpkg-overlays
+if [[ ! -d "${layers}/vcpkg" ]]; then
+  echo "-----> Restore cache from build image"
+  cp -r /layers/cpp/vcpkg-cache "${layers}/vcpkg"
+fi
+cat > "${layers}/vcpkg.toml" <<EOL
+build = true
+cache = true
+launch = false
+EOL
+
+echo "-----> Setup build directory"
+cp -r /layers/cpp/gcf/cmake /var/tmp/source
+cp -r /workspace /var/tmp/source/application
+if [ -r /workspace/vcpkg.json ]; then cp /workspace/vcpkg.json /var/tmp/source; fi
```

Comment:
Instead of an `if` one-liner, consider `test -r <file> && cp <file> /var/tmp/source/`

Reply:
Fixed.